src/components/icons/Icon.jsx

```diff
@@ -195,10 +202,15 @@ export const STD_TYPE = {
   STD_PLUS: 'std-plus',
   STD_POINTS: 'std-points',
   STD_PROFILE: 'std-profile', // new
+  STD_PROFILE_VIEW: 'std-profile_view',
+  STD_QUESTION: 'std-question',
   STD_RECENT_QUESTIONS: 'std-recent_questions', // new
+  STD_RELOAD: 'std-reload',
   STD_REPORT_FLAG: 'std-report_flag',
+  STD_ROTATE: 'std-rotate', //new
   STD_SEARCH: 'std-search',
   STD_SEEN: 'std-seen', // new
+  STD_SETTINGS: 'std-settings', // new
```

Comment:
why do we need this `// new` comment?

Reply:
just for our purposes, (it's new, it means we've never had it before). it will be removed when legacy icons are removed after making a switch.

Reply:
it's better to add meaningful comments, even if that means they will be a bit longer